c_glib/arrow-glib/basic-data-type.h

```diff
@@ -418,6 +418,22 @@ struct _GArrowTemporalDataTypeClass
   GArrowFixedWidthDataTypeClass parent_class;
 };
 
+#define GARROW_TYPE_BINARY_VIEW_DATA_TYPE (garrow_binary_view_data_type_get_type())
+GARROW_AVAILABLE_IN_19_0
+G_DECLARE_DERIVABLE_TYPE(GArrowBinaryViewDataType,
+                         garrow_binary_view_data_type,
+                         GARROW,
+                         BINARY_VIEW_DATA_TYPE,
+                         GArrowDataType)
+struct _GArrowBinaryViewDataTypeClass
+{
+  GArrowDataTypeClass parent_class;
+};
+
+GARROW_AVAILABLE_IN_19_0
+GArrowBinaryViewDataType *
+garrow_binary_view_data_type_new(void);
```

Comment:
Could you move this to https://github.com/apache/arrow/blob/40b2fca4742e2692a917755fd8db2939e10fa02d/c_glib/arrow-glib/basic-data-type.h#L772 ?

Reply:
Thanks. moved.